src/explorer/model/OmittedStatus.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+export interface IOmittedStatus {
+    status: string;
```

Comment:
what if `status` is wrongly set to an arbitary string rather than one of "conflict" "duplicate" "normal" ?